drivers/mipi_dbi/mipi_dbi_spi.c

```diff
@@ -38,6 +42,18 @@ struct mipi_dbi_spi_data {
 #define MIPI_DBI_SPI_READ_REQUIRED DT_INST_FOREACH_STATUS_OKAY(_WRITE_ONLY_ABSENT) 0
 uint32_t var = MIPI_DBI_SPI_READ_REQUIRED;
 
+/* Expands to 1 if the node does reflect the enum in `xfr-min-bits` property */
+#define _XFR_8BITS(n) (DT_INST_PROP(n, xfr_min_bits) == MIPI_DBI_SPI_XFR_8BIT) |
+#define _XFR_16BITS(n) (DT_INST_PROP(n, xfr_min_bits) == MIPI_DBI_SPI_XFR_16BIT) |
```

Comment:
IIRC underscores should not be used, see https://github.com/zephyrproject-rtos/zephyr/issues/9882, maybe just `XFR_.*BITS` or `MIPI_DBI_SPI_XFR_.*BITS`. _WRITE_ONLY_ABSENT should be fixed as well.